Add recent core changes (#600)
owncloud/docs-server: modules/admin_manual/pages/configuration/user/user_roles.adoc

:files_pdfviewer-url: {oc-marketplace-url}/apps/files_pdfviewer
:collabora-online-app-url: {oc-marketplace-url}/apps/richdocuments

:description: ownCloud provides several roles a user can have. This description gives an overview of these roles.

== Introduction

{description}

== User Request for Changing a Role

> A user can request changing is current role. To do so, the user goes into his personal settings view which can be accessed via menu:User Name[Personal > General > Account]. When clicking the button btn:[Request new role], the new role can be entered and an email will be sent to the admin group for further processing. 

Reviewer: ```suggestion
Users can request a change of their current role under menu:User Name[Personal > General > Account]. When clicking the button btn:[Request new role], the new role can be entered and an email will be sent to the admin group for further processing. 
```
